plugins/integration/providers/example/lib/model/IntegrationExampleProvider.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * @package plugins.example
+ * @subpackage model
+ */
+class IntegrationExampleProvider implements IIntegrationProvider
```

Comment:
Please keep this class name as is, but rename ExampleIntegrationProvider to ExampleIntegrationProviderType, these class names are too similar.